config/e3sm/config_grids.xml

```diff
@@ -481,6 +481,16 @@
       <mask>gx1v6</mask>
     </model_grid>
 
+    <model_grid alias="f19_f19_ww3a" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">1.9x2.5</grid>
+      <grid name="lnd">1.9x2.5</grid>
+      <grid name="ocnice">1.9x2.5</grid>
+      <grid name="rof">r05</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">ww3a</grid>
+      <mask>gx1v6</mask>
+    </model_grid>
+
```

Comment:
My recommendation is:
If you are going to add a new grid, how about the one I posted (the ww3a grid)? The only non-stub component in the ADWAV compset is DWAV so ww3 is the most appropriate grid for an ADWAV test.
If that is rejected:
Have you tried this new grid with AQP1?
The whole point of f19_f19 in the first place was that represented a lowest common denominator between models. Given that E3SM does not use the FV dycore (as far as I know), is there any reason to just not remove the null wav entry?
Given the issues we have run into with this test, at a minimum, I think tests should be run on both cesm and e3sm.

Reply:
@gold2718 , thanks for the thoughtful response. I think your first option is a good one.